src/components/FormFields.jsx

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+
+const typeValueMapping = {
+  file: e => e.target.files[0],
+  checkbox: e => e.target.checked,
+  text: e => e.target.value
+};
+
+export const Input = ({
+  id,
+  value,
+  type,
+  name,
+  groupId,
+  onChange = () => {},
+  error,
+  placeholder = "",
+  label,
+  initialChecked,
+  checked,
+  classes: { wrapperClass, inputClass, errorClass },
+  ...rest
+}) => {
+  const isTypePresentInMapping = !!typeValueMapping[type];
```

Comment:
If type is not present in mapping. Give a warning at form init. Also specify the type for which warning is thrown. Also mention that falling back to "text"